infinity/formats/itm_v1.py

```diff
@@ -485,7 +628,7 @@ class ITM_V10_Format (Format):
             { 'key': 'opcode_number',
               'type': 'WORD',
               'off': 0x0000,
-              'enum': 'effects',
+              'enum': 'effects', # FIXME: this ids file does not exist
               'label': 'Opcode number'},
```

Comment:
Do you think that adding logic to the class' constructor that initialises the enum with an appropriate set of values depending on game type would be an acceptable solution?